config/application.rb

```diff
@@ -28,7 +28,23 @@ class Application < Rails::Application
     # config.i18n.load_path += Dir[Rails.root.join('my', 'locales', '*.{rb,yml}').to_s]
     # config.i18n.default_locale = :de
 
+    config.relative_url_root = ENV['RAILS_RELATIVE_URL_ROOT'] || ''
```

Comment:
required for 
config.feedback_url = config.relative_url_root + '/feedback/new'
its all GDS standards